### PR TITLE
Run validate on sgrep-rules during testing

### PR DESCRIPTION
**Pull request by brendongo**

### CODE REVIEW

File: sgrep_lint/tests/run-lint-tests.sh [hunk 1]
```diff
@@ -147,6 +147,7 @@ $SGREP --strict --config=tests/python/good.yaml tests/lint
 #echo TODO: disabled sgrep-rules regression testing for now
 rm -rf /tmp/sgrep-rules && git clone https://github.com/returntocorp/sgrep-rules /tmp/sgrep-rules
 $SGREP --dangerously-allow-arbitrary-code-execution-from-rules --strict --test --test-ignore-todo /tmp/sgrep-rules
```
Comment (ievans) [inline sgrep_lint/tests/run-lint-tests.sh:149]: Why not just replace both these lines with `cd /tmp/sgrep-rules && make test`?

Comment (brendongo) [reply sgrep_lint/tests/run-lint-tests.sh:149]: This runs locally using python while sgrep-rules make test needs a binary executable called sgrep-lint

Comment (ievans) [reply sgrep_lint/tests/run-lint-tests.sh:149]: Should/couldn't we just alias the local python run to $sgrep-lint?

Comment (brendongo) [reply sgrep_lint/tests/run-lint-tests.sh:149]: I tried (not very hard) but couldn't get make to pickup the new alias

**Pull request merged**